spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -81,7 +81,7 @@ This is a helper function to open a new channel.
 This is a safety function in case of a channel closing and the controller chain needs to regain access to an interchain account on the host chain.
 
 ```typescript
-function InitChannel(portId: string, connectionId: string) returns (nil){
+function InitChannel(portId: string, connectionId: string) returns (nil) {
```

Comment:
Can we remove this? We decided not to implement it for the time being in ibc-go and will instead leave it up to ica-auth application developers.